doc/reference/src/core-language.pdc

```diff
@@ -187,9 +187,19 @@ Term judgements have the form:
 Γ ⊢ M \overset{σ}{:} S
 \end{align}
 
-where $σ \in {0,1}$. A judgement with $σ = 0$ constructs a term with no computational content, while a judgement with $σ = 1$ constructs a term which will be computed with. 
+where $σ \in {0,1}$, except for primitive constants.
```

Comment:
Primitive constants are annotated with `gamma`, not `sigma` now right? So `sigma = {0,1}` without exceptions?

Reply:
Yes; the judgement syntax is otherwise the same, but σ is always 0 or 1.

Removed this.